sympy/concrete/summations.py

```diff
@@ -483,7 +483,10 @@ def is_convergent(self):
             return S.false
 
         ### ------------- integral test -------------- ###
-        if is_decreasing(sequence_term, interval):
+        inequality = Derivative(sequence_term).doit() <= 0
+        solution_inequality = solveset(inequality, sym, S.Reals)
+        condition2 = solution_inequality.is_Interval and solution_inequality.right == upper_limit
+        if is_decreasing(sequence_term, interval) or condition2:
```

Comment:
I agree that the use of `is_increasing` is redundant. I can remove it, if you are agree.

I agree also that the solution to `sol = solveset(_, x, Interval(1,oo))` should be `(1, oo)`.

Another way for treating the `sum(log(n)/n**3)` could be adding some comparision test in `summation.py`.  Probably this way is more performing, but it is less general.